locales/English/credits.php

```diff
@@ -57,7 +57,8 @@
 	array(
 		'The webDiplomacy Developers',
 		'<a href="/profile.php?userID=15658">jmo1121109</a>, current lead developer<br />
-		<a href="/profile.php?userID=10">kestasjk</a>'
+		<a href="/profile.php?userID=10">kestasjk</a><br />

```

Comment:
Can you just remove this file from your PR? Aurelin mentioned you weren't fixing this, so I already did, sorry!

Reply:
Yep yep. 
Although, I do have to ask, why change to green? (and not keep the grey, haha)

Reply:
bo_sox48 is working on an optional dark mode, and the green looks better on both than the grey does.